activestorage-openstack.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "fog-openstack", '~> 0.3'
+  s.add_dependency "fog-openstack", '>= 0.3.1', '< 1.0.0'
```

Comment:
needs to be 0.3.1 or >= 1.0.0 . i'd try >= 1.0.0 first but there may be other braking changes /shrug